src/amp-story-player.js

```diff
@@ -36,7 +36,7 @@ const LoadStateClass = {
 
 /** @const {string} */
 const CSS = `
-  :host { all: initial; display: block; border-radius: 0 !important; width: 405px; height: 720px; overflow: auto; }
+  :host { all: initial; display: block; border-radius: 0 !important; width: 320px; height: 480px; overflow: auto; }
```

Comment:
Unrelated suggestion: You can use a separate CSS file that's compiled inline into here, and then you get tooling/etc. See `src/service/video/install-autoplay-styles.js` for example.

Reply:
This part still works:

```js
import {cssText} from '../../../build/video-autoplay.css';
```

Regardless of delivery mechanism you can build the `cssText` string that way to get all other CSS tooling.

Reply:
Got it, will address in a follow-up. Thanks for the pointer!

Reply:
Following up in #26604